solutions/swb-reference/src/services/sshKeyService.ts

```diff
@@ -42,8 +42,44 @@ export default class SshKeyService implements SshKeyPlugin {
   public async listUserSshKeysForProject(
     request: ListUserSshKeysForProjectRequest
   ): Promise<ListUserSshKeysForProjectResponse> {
-    // TODO: implement
-    throw new Error('Method not implemented.');
+    const { projectId, userId } = request;
+
+    // get envMgmtRoleArn and externalId from project record
+    const { envMgmtRoleArn, externalId } = await this._projectService.getProject({ projectId });
```

Comment:
if you pull in `develop`, you should be able to use the helper I made which is `await this._getEnvMgmtRoleArnAndExternalIdFromProject(projectId);`